diff --git a/src/actions.cpp b/src/actions.cpp
index 77cc295..3e0ca06 100644
--- a/src/actions.cpp
+++ b/src/actions.cpp
@@ -540,6 +540,11 @@ void CAction::setRTPStreamActInfo(const char* P_value)
              M_rtpstream_actinfo.ticks_per_packet= 160;
              break;
 
+    case 98: M_rtpstream_actinfo.ms_per_packet= 30;
+             M_rtpstream_actinfo.bytes_per_packet= 50;
+             M_rtpstream_actinfo.ticks_per_packet= 240;
+             break;
+
     default: M_rtpstream_actinfo.ms_per_packet= -1;
              M_rtpstream_actinfo.bytes_per_packet= -1;
              M_rtpstream_actinfo.ticks_per_packet= -1;

